tests/installation/boot_encrypt.pm

```diff
@@ -16,10 +16,11 @@ use testapi qw(check_var get_var record_info);
 use version_utils qw(is_leap is_sle is_leap_micro is_sle_micro is_alp);
 
 sub run {
-    # With newer grub2 (in TW only currently), entering the passphrase in GRUB2
+    # With newer grub2 (in TW and SLE 15SP6 currently), entering the passphrase in GRUB2
     # is enough. The key is passed on during boot, so it's not asked for
     # a second time.
-    return if is_boot_encrypted && !is_leap && !is_sle && !is_leap_micro && !is_sle_micro && !is_alp;
+    my $need_to_enter_passphrase = is_leap || is_sle('<15-sp6') || is_leap_micro || is_sle_micro || is_alp;
```

Comment:
Technically `check_var('STAGING', 'V')` only. Or is it ok to break the main SP6 group and other stagings?

Reply:
@g7 ^

Reply:
Yes, it's ok for us (SP6) to break the other stagings

Reply:
Also the main group, i.e. no new build will pass until Staging:V is accepted?

Reply:
yes, temporary breaking the main tests is okay with us as well